zerver/lib/email_notifications.py

```diff
@@ -679,18 +679,19 @@ def handle_missedmessage_emails(
         )
 
 
-def followup_day2_email_delay(user: UserProfile) -> timedelta:
-    days_to_delay = 2
+def followup_email_delay(user: UserProfile, *, days: int) -> timedelta:
+    days_to_delay = days
```

Comment:
We probably should include an assert that `days >= 2` -- at least with the current algorithm, the delay function will potentially generate a delay of 0 with certain days of the week.

The other thing we'll want to be careful with in this API is that you can't do 

first_email_date = followup_email_delay(user, 2)
second_email_date = followup_email_delay(user, 3)

in that you may have both of those emails go out on a Friday.

Idunno, maybe the assert is the wrong model and we do a larger refactor here.

Reply:
Yeah, maybe CZO is a better place to figure this out. Omar may have feedback as well.

Reply:
Updated for the refactor to schedule these onboarding emails as a set so that we can take in to account the whole flow of when theses emails are sent. This eliminates the day 1 issue for now, but allows us to adjust for that if we did ever want to add a day after registering email.